controller/eks-cluster-config-handler.go

```diff
@@ -371,7 +388,39 @@ func createStack(svc *cloudformation.CloudFormation, name string, displayName st
 	return stack, nil
 }
 
+func validateUpdate(config *v13.EKSClusterConfig) error {
+	clusterVersion, err := semver.New(fmt.Sprintf("%s.0", config.Spec.KubernetesVersion))
+	if err != nil {
+		return fmt.Errorf("improper version format for cluster [%s]: %s", config.Name, config.Spec.KubernetesVersion)
+	}
+
+	var errors []string
+	// validate nodegroup versions
+	for _, ng := range config.Spec.NodeGroups {
+		version, err := semver.New(fmt.Sprintf("%s.0", aws.StringValue(ng.Version)))
+		if err != nil {
+			errors = append(errors, fmt.Sprintf("improper version format for nodegroup [%s]: %s", ng.NodegroupName, aws.StringValue(ng.Version)))
+		}
+		if clusterVersion.EQ(*version) {
+			continue
+		}
+		if clusterVersion.Minor-version.Minor == 1 {
```

Comment:
wouldn't this be 1 or 0 ? If they are on same version would be fine ?

Reply:
that what the EQ above this is checking